apps/emqx/src/emqx_schema.erl

```diff
@@ -73,6 +73,8 @@
 -export([conf_get/2, conf_get/3, keys/2, filter/1]).
 -export([ssl/1]).
 
+-define(RESOLVE_TIMEOUT, timer:seconds(10)).
```

Comment:
not entirely sure if it is a good idea to resolve hostname when parsing schema.
DNS resolution may fail from time to time.

i.e. it might not always be a misconfiguration when a host is not resolvable,
for example, when deploying a service, its DNS is not yet replicated to EMQ X node.

Reply:
this macro is no longer needed.